lib/vyasa/written/verse.ex

```diff
@@ -3,20 +3,18 @@ defmodule Vyasa.Written.Verse do
   import Ecto.Changeset
 
   alias Vyasa.Written.{Source, Chapter, Translation}
-  alias Vyasa.Sangh.{Comment}
-  alias Vyasa.Adapters.Binding
 
   @primary_key {:id, Ecto.UUID, autogenerate: true}
   schema "verses" do
     field :no, :integer
     field :body, :string
     field :binding, :map, virtual: true
+    field :comments, {:array, :map}, virtual: true, default: []
```

Comment:
:map is because of ecto has a technical limitation

ideally we'd have put array of comments, but ecto doesn't support that and since structs are maps, then this will work